### DISCUSSION TITLE
测试waline评论功能时出错

### DISCUSSION THREAD

**Discussion by 123de234** [hugo-next/hugo-theme-next#61] [Q&A]
Bug 如图：

<img width="1280" alt="issue" src="https://user-images.githubusercontent.com/23733252/193410589-ffe22bdb-9b0c-4664-bc59-5412e5a17144.png">

yml设置如下：
<img width="359" alt="屏幕截图 2022-10-01 205640" src="https://user-images.githubusercontent.com/23733252/193410613-c6c45a44-f640-4572-a451-03a08d4b8a9b.png">

**Comment by elkan1788** [comment]
这里你的地址配置错了，只需要 `/ui/loginxxxxx` 前面那个地址就可以啦。

> 另外这类使用问题建议直接在 `Discussions` 讨论区进行提问，谢谢。
